docker-compose.dev.yml

```diff
@@ -0,0 +1,59 @@
+# Run options:
+# 1. There are no options - This add additional tools to aid in the development of Libre Photos
+#   run cmd: docker-compose up -f docker-compose.yml -f docker-compose.dev.yml -d
+# 2. Current added tools:
+#    pgadmin User admin@admin pass admin port 3001
+
+# DO NOT EDIT
+# The .env file has everything you need to edit.
+# Run options:
+# 1. Use prebuilt images (preferred method):
+#   run cmd: docker-compose up -d
+# 2. Build images on your own machine:
+#   build cmd: COMPOSE_DOCKER_CLI_BUILD=1 DOCKER_BUILDKIT=1 docker-compose build
+#   run cmd: docker-compose up -d
+
+version: '3.8'
+services:
+
+  frontend:
+    tty: true
+    environment:
+    - DEBUG=1
+    build:
+      context: .
+      dockerfile: frontend/Dockerfile
+    volumes:
+      - ${codedir}/frontend:/usr/src/app
```

Comment:
This should be the name of the repository "librephotos-frontend"

Reply:
You mean frontend should be renamed to librephotos-frontend?

Reply:
Yes, that's what I want :+1: